aiida_common_workflows/workflows/relax/cp2k/generator.py

```diff
@@ -152,6 +152,16 @@ def define(cls, spec):
         The ports defined on the specification are the inputs that will be accepted by the ``get_builder`` method.
         """
         super().define(spec)
+        spec.input(
+            'protocol',
+            valid_type=ChoiceType((
+                'fast', 'moderate', 'precise', 'verification-PBE-v1-DZVP-GTH', 'verification-PBE-v1-DZVP-GTH-rev',
+                'verification-PBE-v1-TZV2P-GTH'
```

Comment:
Should we also no have `verification-pbe-v1` here, for consistency with all the others? Could just function as an alias if it really points to one of the specific versions you already have added here. I take it the verification study was run with just one of these protocols? Or do you need  all of them depending on the structure you are running?

Reply:
I will double-check, but I guess the protocol that was used for the final calculation is `verification-PBE-v1-TZV2P-GTH` -  I can rename it to "verification-pbe-v1".

The other two "verification-PBE.." could probably be removed. They were used while testing different parameters.